### PR TITLE
feat(wrw): update custody wallet recovery flow

### PR DESCRIPTION
**Pull request by mohitkh7**
Ticket: COIN-597

Updated waller recovery form for custody wallet. It changes the form to consist of only 5 input fields.

### CODE REVIEW

File: src/containers/EvmCrossChainRecoveryWallet/EvmCrossChainRecoveryBaseForm.tsx [hunk 1]
```diff
@@ -11,10 +11,12 @@ import { BitgoEnv, safeEnv } from '~/helpers';
 
 export function EvmCrossChainRecoveryBaseForm({
   formik,
+  isCustodyWallet,
 }: {
   formik: {
     setFieldValue: (a: string, b: string) => void;
   };
+  isCustodyWallet: boolean;
```
Comment (mullapudipruthvik) [inline src/containers/EvmCrossChainRecoveryWallet/EvmCrossChainRecoveryBaseForm.tsx:19]: can't we get this using useParams<'env' | 'wallet'> ?

File: src/containers/EvmCrossChainRecoveryWallet/EvmCrossChainRecoveryBaseForm.tsx [hunk 2]
```diff
@@ -111,30 +117,37 @@ export function EvmCrossChainRecoveryBaseForm({
...
+        <div className="tw-mb-4">
+          <FormikTextfield
+            HelperText='"Tip" to the Polygon miner. This is by default 10 Gwei.'
+            Label="Max Priority Fee Per Gas (Gwei)"
+            name="maxPriorityFeePerGas"
+            Width="fill"
+          />
+        </div>
+      )}
       <div className="tw-mb-4">
```
Comment (mullapudipruthvik) [inline src/containers/EvmCrossChainRecoveryWallet/EvmCrossChainRecoveryBaseForm.tsx:151]:
```suggestion
      {!isCustodyWallet && (
        <div className="tw-mb-4">
          <FormikTextfield
            HelperText="Gas limit for the Polygon transaction. The value should be between 30,000 and 20,000,000. The default is 500,000 unit of gas."
            Label="Gas Limit"
            name="gasLimit"
            Width="fill"
          />
        </div>
 
        <div className="tw-mb-4">
          <FormikTextfield
            HelperText="Max fee per gas for the Polygon transaction. The default is 20 Gwei."
            Label="Max Fee Per Gas (Gwei)"
            name="maxFeePerGas"
            Width="fill"
          />
        </div>
  
     
        <div className="tw-mb-4">
          <FormikTextfield
            HelperText='"Tip" to the Polygon miner. This is by default 10 Gwei.'
            Label="Max Priority Fee Per Gas (Gwei)"
            name="maxPriorityFeePerGas"
            Width="fill"
          />
        </div>
      )}
      <div className="tw-mb-4">
```

**Review by mullapudipruthvik — COMMENTED** [review]
Please add some screen shots as well

File: src/containers/EvmCrossChainRecoveryWallet/EvmCrossChainRecoveryBaseForm.tsx (hunk 1 above)
Comment (mohitkh7) [reply src/containers/EvmCrossChainRecoveryWallet/EvmCrossChainRecoveryBaseForm.tsx]: Started using useParams <'wallet'>

File: src/containers/EvmCrossChainRecoveryWallet/EvmCrossChainRecoveryBaseForm.tsx (hunk 2 above)
Comment (mohitkh7) [reply src/containers/EvmCrossChainRecoveryWallet/EvmCrossChainRecoveryBaseForm.tsx]: As suggested, combined 3 different condition into 1.

**Comment by mohitkh7** [comment]
https://github.com/BitGo/wallet-recovery-wizard/assets/19525624/a078b4ce-f8ee-4b58-bf21-62e6c32200e3

Local demo screen recording

**Review by mullapudipruthvik — APPROVED** [review]

**Pull request merged**